src/components/entity/state-badge.html

```diff
@@ -75,6 +75,8 @@
     if (newVal.attributes.entity_picture) {
       hostStyle.backgroundImage = 'url(' + newVal.attributes.entity_picture + ')';
       iconStyle.display = 'none';
+    } else if (newVal.attributes.icon_color) {
+      iconStyle.color = `${newVal.attributes.icon_color}`;
```

Comment:
no need for the template

Reply:
my goal was to cast/force it to string, so we don't break stuff.